Jenkins/InedoExtension/JenkinsArtifactImporter.cs

```diff
@@ -52,25 +52,25 @@ public async Task<string> ImportAsync()
 
             try
             {
-                this.Logger.LogInformation($"Importing {this.ArtifactName} from {this.JobName}...");
-                var client = new JenkinsClient(this.ConnectionInfo, this.Logger);
+                Logger.LogInformation($"Importing {ArtifactName} from {JobName}...");
+                var client = new JenkinsClient(ConnectionInfo, Logger);
 
                 zipFileName = Path.GetTempFileName();
-                this.Logger.LogDebug("Temp file: " + zipFileName);
+                Logger.LogDebug("Temp file: " + zipFileName);
 
-                this.Logger.LogDebug("Downloading artifact...");
-                await client.DownloadArtifactAsync(this.JobName, jenkinsBuildNumber, zipFileName).ConfigureAwait(false);
-                this.Logger.LogInformation("Artifact downloaded.");
+                Logger.LogDebug("Downloading artifact...");
+                await client.DownloadArtifactAsync(JobName, jenkinsBuildNumber, zipFileName, SubFolder).ConfigureAwait(false);

```

Comment:
Apart from cleaning up the use of 'this', this line and creating the property itself were the only real changes to this class.